### PR TITLE
Directory: Bind get_current_drive() method

### PR DESCRIPTION
**Pull request by neikeq**
:hand:

### CODE REVIEW

File: core/bind/core_bind.cpp [hunk 1]
```diff
@@ -431,7 +431,7 @@ String _OS::get_latin_keyboard_variant() const {
 
 String _OS::get_model_name() const {
 
-    return OS::get_singleton()->get_model_name();
+	return OS::get_singleton()->get_model_name();
```
Comment (neikeq) [inline core/bind/core_bind.cpp]: ok..

Comment (akien-mga) [inline core/bind/core_bind.cpp]: Fine by me :)

**Review by akien-mga — APPROVED** [review]

File: core/bind/core_bind.cpp [hunk 2]
```diff
@@ -1863,6 +1863,10 @@ String _Directory::get_drive(int p_drive){
 	ERR_FAIL_COND_V(!d,"");
 	return d->get_drive(p_drive);
 }
+int _Directory::get_current_drive() {
+	ERR_FAIL_COND_V(!d,"");
```
Comment (neikeq) [inline core/bind/core_bind.cpp]: i'm a moron

Comment (bojidar-bg) [inline core/bind/core_bind.cpp]: What an int! :laughing:

**Pull request merged**